model/src/common/search/index.ts

```diff
@@ -0,0 +1,17 @@
+import Joi from 'joi'
+
+import { type SearchOptions } from '~/src/common/search/types.js'
+
+/**
+ * Field definitions for search options.
+ */
+export const searchOptionFields = {
+  title: Joi.string().trim().min(1).optional()
```

Comment:
Can I double check what "optional" means in a GET sense?

Think we also need `.allow('')` for when `?title=` is an empty string

Reply:
That's a good spot! Joi wouldn't let an empty string through- thought it would as it's optional

Reply:
I think we can just remove `min(1)`

Reply:
Nope I was wrong 😅